rwlib/source/rw/defines.hpp

```diff
@@ -19,6 +19,6 @@
 	(void) var
 
 #define RW_UNIMPLEMENTED(msg) \
-	RW_MESSAGE("Unimplemented" << msg)
+	RW_MESSAGE("Unimplemented: " << msg)
```

Comment:
nit: more descriptive commit name next time so we know if this is to fight a regression OR a style fix OR just a visual fix